### PR TITLE
feat: extra vaults monitoring

### PR DESCRIPTION
**Pull request by madlabman**

### CODE REVIEW

File: ethereum-operations/src/subagents/vaults/agent-vaults.ts [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+  return findings;
+}
+
+async function handleBurnerSharesTx(
+  txEvent: TransactionEvent,
+  findings: Finding[]
+) {
+  const events = txEvent
+    .filterLog(TRANSFER_SHARES_EVENT, LIDO_ADDRESS)
+    .filter((e) => e.args.from.toLowerCase() === BURNER_ADDRESS.toLowerCase());
```
Comment (dgusakov) [inline ethereum-operations/src/subagents/vaults/agent-vaults.ts:259]: So we don't expect any stETH transfers from the Burner, right? What happens when stETH is burned? Isn't it transfer to 0x0000..00?

**Review by dgusakov — APPROVED** [review]

File: ethereum-operations/src/subagents/vaults/constants.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+export const TRANSFER_SHARES_EVENT = `
+    event TransferShares(
+        address indexed from,
+        address indexed to,
+        uint256 sharesValue
+    )
+`;
+
+export const ETH_DISTRIBUTED_EVENT = `
```
Comment (vgorkavenko) [inline ethereum-operations/src/subagents/vaults/constants.ts:23]: Unused var

File: ethereum-operations/src/subagents/vaults/agent-vaults.ts [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+  if (withdrawalVaultBalanceDiff.gte(WITHDRAWAL_VAULT_BALANCE_DIFF_INFO)) {
+    const severity = withdrawalVaultBalanceDiff.gte(
+      WITHDRAWAL_VAULT_BALANCE_DIFF_HIGH
+    )
+      ? FindingSeverity.High
+      : FindingSeverity.Info;
+
+    findings.push(
+      Finding.fromObject({
+        name: "Withdrawal Vault Balance significant change",
```
Comment (vgorkavenko) [inline ethereum-operations/src/subagents/vaults/agent-vaults.ts:111]: Pls, add an emoji for each alert, as usual

Comment (madlabman) [reply ethereum-operations/src/subagents/vaults/agent-vaults.ts:111]: 😦

File: ethereum-operations/src/subagents/vaults/constants.ts (hunk 2 above)
Comment (madlabman) [reply ethereum-operations/src/subagents/vaults/constants.ts]: Removed.

File: ethereum-operations/src/subagents/vaults/agent-vaults.ts (hunk 3 above)
Comment (madlabman) [reply ethereum-operations/src/subagents/vaults/agent-vaults.ts]: Done.

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**